falco/templates/daemonset.yaml

```diff
@@ -57,6 +57,7 @@ spec:
             - --cri
             - /run/containerd/containerd.sock
             {{- end }}
+            - --disable-cri-async # @kris-nova default to sync falco mode to eliminate syscall drops
```

Comment:
Instead of hard-coding it here, would it be more appropriate to let the user set this flag via `Values.extraArgs` (on line 66)?  Seems like the change here should be a documentation change that informs the user about the flag rather than a hard-coded value.